prow/config/config_test.go

```diff
@@ -2606,3 +2607,682 @@ func TestRefGetterForGitHubPullRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestSetDecorationDefaults(t *testing.T) {
```

Comment:
Very thorough test coverage, cool!

Personally I'd like to see one or two additional tests exercising the interaction of "*"/org/repo levels within the `DefaultDecorationConfigs` itself. For example, if you have defaults for **both** `org` and `org/repo`, you get the defaults for the latter and not the former. I think it's enough to test that `org` overrides `*` and `org/repo` overrides `org`.

Reply:
addressed.test cases added.